src/libs/AxelarQueryAPI.ts

```diff
@@ -281,6 +280,10 @@ export class AxelarQueryAPI {
       throw new Error("Failed to estimate gas fee");
     }
 
+    if (BigNumber.from(gasLimit).lt(21000)) {
```

Comment:
just do a non zero check? The error is confusing otherwise, if user sends over 21000, is the gas limit not low anymore? Something like `Gas limit must be provided` indicates the purpose of the check better I think